weighing.py

```diff
@@ -186,8 +200,8 @@ def on_change_with_crop(self):
     def get_parcel(self):
         if not self.plantations:
             return
-        plantation = self.plantations[0]
-        if not plantation.parcels:
+        plantation = self.plantations[0].plantation
```

Comment:
crash if self.plantations is None

Reply:
Does not prevent it the previous line?
        if not self.plantations:
            return